src/index.ts

```diff
@@ -470,6 +470,7 @@ export interface OpenWindowParameters {
  */
 export interface InitResult {
   error?: SidebarError;
+  properties?: { [key: string]: any; }
```

Comment:
@mscc-sascha Should we call the properties something specific? Like "clientProperties" or "intergrationProperties" or something else?